step_setup_cloning_env.go

```diff
@@ -0,0 +1,111 @@
+package main
+
+import (
+	"github.com/mitchellh/multistep"
+	"github.com/hashicorp/packer/packer"
+	"context"
+	"github.com/vmware/govmomi"
+	"github.com/vmware/govmomi/find"
+	"github.com/vmware/govmomi/vim25/mo"
+	"fmt"
+	"net/url"
+	"github.com/vmware/govmomi/object"
+)
+
+type StepSetupCloningEnv struct{
+	config *Config
+}
+
+func (s *StepSetupCloningEnv) Run(state multistep.StateBag) multistep.StepAction {
```

Comment:
`finder` object will be also required if we move hardware configuration into separate step. but it can be published right in the cloning step